libafl_bolts/src/llmp.rs

```diff
@@ -1922,6 +1922,31 @@ where
     shmem_provider: SP,
 }
 
+#[inline]
+fn binary_search<SP: ShMemProvider>(
```

Comment:
Looks like we could also just use this method from the stdlib (TIL):
https://doc.rust-lang.org/std/primitive.slice.html#method.binary_search_by